src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -114,14 +117,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $builder->setTimeout(null);
         $process = $builder->getProcess();
 
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
-            $process->disableOutput();
+        try {
+            if (OutputInterface::VERBOSITY_NORMAL > $output->getVerbosity()) {
+                $process->disableOutput();
+            } else {
+                $process->setTty(true);
+            }
+            $process->run();
+        } catch (RuntimeException $e) {
```

Comment:
couldn't runtime exceptions happen for other reasons than unsupported tty mode during ``$process->run();``, causing weird cases of running the process twice ?